src/client/dfs/obj.c

```diff
@@ -412,6 +412,12 @@ open_stat(dfs_t *dfs, dfs_obj_t *parent, const char *name, mode_t mode, int flag
 
 out:
 	if (rc == 0) {
+		if (flags & O_CREAT) {
+			DFS_OP_STAT_INCR(dfs, DOS_CREATE);
```

Comment:
i  think it would be better to not have a CREATE metric, but re-use the mkdir (because that is what open with O_CREAT for dir is) and File_Create or just CREATE for creating a file.